site/src/measure.njk

```diff
@@ -0,0 +1,165 @@
+```
+
+The script instructs WPT to open the [unoptimized demo](https://mini-ecomm.glitch.me/) and simulate a click on the first product of the listing. Metrics are captured only for the product page. Here is the [resultiing test](https://www.webpagetest.org/result/191103_TM_e68d81788d8744762301b44c6e3e72d2/).
+
+Repeat the process on [the demo](https://mini-ecomm-quicklink.glitch.me/) that uses Quicklink. The script looks like:
+
+```
+logData 0
+navigate https://mini-ecomm.glitch.me/
+logData 1
+execAndWait document.querySelector('a').click()
+```
+
+Here is the [resultiing test](https://www.webpagetest.org/result/191103_E3_f8217e45ad837ac084868d4f3b9a4a73/).
+
+The following table compares the main metrics obtained for each of the sites:
+
+{% endmarkdownConvert %}
+
+<img class="article-image" src="/assets/images/screenshots/wpt-metrics-comparison.png" height="105px" width="700px">
+
+{% markdownConvert %}
+
+Next, create a comparison between the tests.
+
+- Open the [WPT test](https://www.webpagetest.org/result/191103_TM_e68d81788d8744762301b44c6e3e72d2/) for the unoptimized site.
+- Click on the median run, that appears in the "First View" cell of the report.
+- Repeat the same process in the [optimized test](https://www.webpagetest.org/result/191103_E3_f8217e45ad837ac084868d4f3b9a4a73/).
+
+To create a comparison test, you need to append the IDs from the previous links as comma separated valuees, and send them as query params to `https://www.webpagetest.org/video/compare.php`:
+
+```
+https://www.webpagetest.org/video/compare.php?tests=test_id_1,test_id_2
+```
+
+The resulting comparison of the test ran previously can be found [here](https://www.webpagetest.org/video/compare.php?tests=191103_TM_e68d81788d8744762301b44c6e3e72d2-r%3A8-c%3A0%2C191103_E3_f8217e45ad837ac084868d4f3b9a4a73-r%3A7-c%3A0&thumbSize=200&ival=500&end=visual).
+
+### Visual Comparison
+
+The unoptimized site starts rendering approximately at **2.5s**, the demo that uses Quicklink, starts at **1.2s**.
+
+{% endmarkdownConvert %}
+
+<img class="article-image" src="/assets/images/screenshots/wpt-visual-comparison.png" height="430px" width="820px">
+
+{% markdownConvert %}
+
+### Video
+
+A video can be generated from the [comparison page](https://www.webpagetest.org/video/compare.php?tests=191103_TM_e68d81788d8744762301b44c6e3e72d2-r%3A8-c%3A0%2C191103_E3_f8217e45ad837ac084868d4f3b9a4a73-r%3A7-c%3A0&thumbSize=200&ival=500&end=visual), by clicking on **Create Video**.
+
+{% endmarkdownConvert %}
+
+<img class="article-image" src="/assets/images/screenshots/wpt-video-comparison.gif" height="435px" width="600px">
+
+{% markdownConvert %}
+
+### Using RUM (Real user monitoring) tools
+
+RUM tools, let you visualize how different metrics evolve in time for real users. If prefetching affects a large amount of pages, you might be able to see more page loads being loaded faster after implementing it, which can be reflected in metrics First Contentful Paint.
```

Comment:
This is currently a long paragraph... :) Could we split this into 2-3 paragraphs to break it up just a little for readability?